diff --git a/README.md b/README.md
index 77fbd73..7873042 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,8 @@
 # WELCOME TO GARBO SEARCH
 It's a garbage lil search tool
 
+If you include this line in your commit, you weren't paying attention!
+
 ## Install Steps:
 * Open VS Code
     * CMD+Shift+P: Install 'Code' command in path
@@ -39,3 +41,4 @@ It's a garbage lil search tool
 * Support searching multiple fields
 * Support only getting the first n results
 * ~~Print results in csv~~
+* ~~Negative search matching~~
diff --git a/main.go b/main.go
index f23b892..a01d492 100644
--- a/main.go
+++ b/main.go
@@ -35,6 +35,7 @@ func main() {
 	for idx, row := range rows {
 		if idx == 0 {
 			headers = row
+			continue
 		}
 		person := make(map[string]string)
 		for jdx, cell := range row {
@@ -44,7 +45,11 @@ func main() {
 	}
 
 	for _, person := range people {
-		if person[searchField] == searchValue {
+		if searchValue[0] == "-"[0] {
+			if "-"+person[searchField] != searchValue {
+				fmt.Printf("%s,%s,%s,%s\n", person["first"], person["last"], person["city"], person["state"])
+			}
+		} else if person[searchField] == searchValue {
 			fmt.Printf("%s,%s,%s,%s\n", person["first"], person["last"], person["city"], person["state"])
 		}
 	}
